### PR TITLE
Add pollTimeout to options

### PR DESCRIPTION
**Pull request by m4hdyar**
Hi,

Added pollTimeout to options and also readme.
Regarding #192.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Example:  `/`
 
 Type: `string`
 
+
```
Comment (christian-bromann) [inline README.md:80]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -91,6 +92,13 @@ Example:  `localhost`
 
 Type: `string`
 
+### pollTimeout
+The startup timeout in ms, it checks if the port is open before starting ChromeDriver Server and then checks again if the chromedriver port is closed after starting ChromeDriver Server.
+
+Example: 10000
```
Comment (christian-bromann) [inline README.md:98]:
```suggestion
Example: `10000`
```

File: README.md [hunk 3]
```diff
@@ -91,6 +92,13 @@ Example:  `localhost`
 
 Type: `string`
 
+### pollTimeout
+The startup timeout in ms, it checks if the port is open before starting ChromeDriver Server and then checks again if the chromedriver port is closed after starting ChromeDriver Server.
```
Comment (christian-bromann) [inline README.md:96]:
```suggestion
The startup timeout in ms, it checks if the port is open before starting ChromeDriver and then checks again if the it is closed after starting it.
```

**Comment by christian-bromann** [comment]
Seems like some tests still fail.

**Comment by m4hdyar** [comment]
> Seems like some tests still fail.

Yeah had forgotten to edit the tests also. I also separated the rest of the default values from the test cases.

**Comment by m4hdyar** [comment]
@christian-bromann Can you please merge this pull request ? or should I do something else ? Because I want to create another pull request in electron service as well regarding to this issue and I'm waiting for this pull request.

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Pull request merged**